app/presenters/two_factor_authentication/selection_presenter.rb

```diff
@@ -120,16 +120,8 @@ def setup_info(type)
         t('two_factor_authentication.two_factor_choice_options.auth_app_info')
       when 'backup_code'
         t('two_factor_authentication.two_factor_choice_options.backup_code_info')
-      when 'phone'
```

Comment:
This code is unreachable, since `PhoneSelectionPresenter` overrides the `info` method and therefore `setup_info` would not be called for those methods. The code here is incorrect anyways, since it's showing the Kantara messaging based on `select_multiple_mfa_options` (should only be shown with `kantara_2fa_phone_restricted`).

https://github.com/18F/identity-idp/blob/dc20140c89771ae1fe431c88342f1f5a95dfab54/app/presenters/two_factor_authentication/phone_selection_presenter.rb#L15-L20